### PR TITLE
executor: refine query metric error message

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs <crazycs520@gmail.com>

### What problem does this PR solve? 

Change query metric error message.

Before this PR

```sql
METRICS_SCHEMA> select * from up;
(1105, u'server_error: server error: 500')
```

This PR

```sql
METRICS_SCHEMA> select * from up;
(1105, u"query metric error, msg: server error: 500, detai: metric storage doesn't set\n")
```

### CODE REVIEW

File: executor/metrics_reader.go [hunk 1]
```diff
@@ -85,7 +85,10 @@ func (e *MetricRetriever) retrieve(ctx context.Context, sctx sessionctx.Context)
 			time.Sleep(100 * time.Millisecond)
 		}
 		if err != nil {
-			return nil, err
+			if err1, ok := err.(*promv1.Error); ok {
+				return nil, errors.Errorf("query metric error, msg: %v, detai: %v", err1.Msg, err1.Detail)
```
Comment (zimulala) [inline executor/metrics_reader.go]: s/detai/detail ?

Comment (crazycs520) [reply executor/metrics_reader.go]: great, done.

**Comment by reafans** [comment]
lgtm

**Pull request merged**